app/assets/javascripts/galleries/add_existing.js

```diff
@@ -47,7 +47,7 @@ $(document).ready(function() {
 });
 
 $(window).on('beforeunload', function() {
-  if (skipWarning || imageIds.length === 0) return;
+  if (skipWarning || imageIds.length === 0) { return false; }
```

Comment:
Not clear on the motivation here (either for the braces or the false value). What was the original complaint?

Reply:
complaint was about returning both a value and not returning something from a function, brackets are because I find that more readable on javascript ifs. ...I think that's the one I was most concerned it would break something tho.

Reply:
Yeah, events like this are always awkward and returning a string is even now deprecated (but the alternatives aren't well supported). Returning false might do something weird in some browsers. I'd leave this and ignore the linter on it.

Reply:
(also FWIW I never see braces on single-line if statements in JavaScript - it's actually kinda distracting to me)